cypress/integration/search_page_spec.js

```diff
@@ -0,0 +1,207 @@
+const sortSelectBox = '[data-testid=sort-select]'
+const datasetsFoundCount = '[data-testid=result-count]'
+const paginator = '[data-testid=paginator]'
+const dialogContent = '[data-testid=dialog-content]'
+const search = '[data-testid=search]'
+const datasets = '[data-testid=dataset-list]'
+const firstDataset = '[data-testid=dataset-list] > :nth-child(1)'
+const apiAccessibleCheckbox = '[data-testid=api-accessible-checkbox]'
+const organizations = '[data-testid=facet-sidebar] > :nth-child(1)'
+const cogoCheckBox = '[data-testid=facet-sidebar] > :nth-child(1) > :nth-child(2) > .checkbox-indicator'
+const keywords = '[data-testid=facet-sidebar] > :nth-child(2)'
+const bicycleCheckBox = '[data-testid=facet-sidebar] > :nth-child(2) > :nth-child(2) > .checkbox-indicator'
```

Comment:
(Tony) I thought we were still planning to add a testid for each of the facet lists?

Reply:
Agreed I will update.